app/javascript/components/tree-view/reducers/index.js

```diff
@@ -1,20 +1,22 @@
 import { ActionTypes } from '@manageiq/react-ui-components/dist/wooden-tree';
 import onCheck from './oncheck';
 import onClick from './onclick';
-import others from './others';
+import { checkAll, checkSilent } from './others';
 
 export const ACTIONS = {
   ...ActionTypes,
   EMPTY_TREE: 'tree.empty',
   CHECK_ALL: 'tree.checkAll',
+  CHECKED_SILENT: 'node.checked.silent',
 };
 
 const reducers = (oncheck, onclick, treeName) => (state = {}, action) => {
   switch (action.type) {
     case ACTIONS.CHECKED: return onCheck(state, action, oncheck, treeName);
     case ACTIONS.SELECTED: return onClick(state, action, onclick);
     case ACTIONS.EMPTY_TREE: return {};
-    case ACTIONS.CHECK_ALL: return others.checkAll(state, action.value, treeName);
+    case ACTIONS.CHECK_ALL: return checkAll(state, action.value, treeName);
+    case ACTIONS.CHECKED_SILENT: return checkSilent(state, action);
```

Comment:
it's inconsistent: check vs check**ed**

Reply:
Different action: 
* I send an action to check all nodes.
* I checked a node so please make the changes.

However, if you persist I don't mind changing it

Reply:
case ACTIONS.CHECK**ED**_SILENT: return checkSilent(state, action);

Reply:
It's not about the two separate actions, but inconsistency on a single line. Constant vs Function name.